client/components/upstreamMessages/html/src/recommended/index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Upstream Messages - Recommended Integration - PayPal Web SDK</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <h1>Upstream Messages Recommended Integration</h1>
+
+    <div id="message-background">
+      <paypal-message id="paypal-message"></paypal-message>
```

Comment:
Should credit offer details be available on page load since a quantity of `1` is the default?
<img width="667" alt="Screenshot 2025-06-11 at 11 45 14 AM" src="https://github.com/user-attachments/assets/82eab691-a5e6-4cf4-85ae-b74962bd3006" />